docs/hybrid-slurm-cluster/hybrid-slurm.md

```diff
@@ -0,0 +1,423 @@
+# Setting up Hybrid Slurm Clusters Using the HPC Toolkit
+
+## Introduction
+
+This document intends to support the use of the hybrid slurm terraform modules
+provided by SchedMD via [Slurm on GCP][slurm-gcp] and are availabe in the HPC
+Toolkit through the [schedmd-slurm-gcp-v5-hybrid][hybridmodule] module.
+
+Cloud hybrid slurm clusters are slurm clusters that manage both local and cloud
+partitions, where cloud partitions can elastically create resources in Google
+Cloud as needed.
+
+> **_NOTE:_** Since on-premise Slurm configurations can vary significantly,
+> it is likely that this document does not cover every edge case.
+> The intent is to provide a useful starting point for setting up cloud
+> hybrid partitions with Slurm, the HPC Toolkit and Google Cloud.
+
+## About the Hybrid Configuration Module
+
+The [schedmd-slurm-gcp-v5-hybrid][hybridmodule] module relies on the
+[slurm\_controller\_hybrid] terraform module in [Slurm on GCP][slurm-gcp]. The
+module defines a configuration for a hybrid environment using the Slurm
+on GCP scripts for deploying cloud hybrid compute resources. This includes:
+
+* Creating a configuration directory with:
+  * a `cloud.conf` file that extends the `slurm.conf` configuration.
+  * a `cloud_gres.conf` file that extends the `gres.conf` configuration.
+  * the [slurm-gcp] resume, suspend and synchronization scripts for
+    the hybrid partitions.
+* Compute instance templates that describe the nodes created by each cloud
+  hybrid partition.
+* Metadata in the target cloud project that informs newly created cloud compute
```

Comment:
nit: remove the word `target`. I don't think it is adding anything.

Reply:
Done